corehq/apps/userreports/tests/test_export.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+from django.test import SimpleTestCase
+
+from .test_data_source_config import get_sample_data_source
+from ..sql import get_indicator_table
+from ..views import process_url_params
+
+
+class ParameterTest(SimpleTestCase):
```

Comment:
So the format is `<field-name>-<directive>` for the key, then the parameter will be parsed by the corresponding function.  Happy to hear alternative suggestions.